understanding/headings-and-labels.html

```diff
@@ -58,190 +56,133 @@
+                  <dd>特に認知障害又は視覚障害のある人々が、ページ上の自分の位置を確認することができる。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to help users understand what information
-                  is contained in Web pages and how that information is organized. When headings are
-                  clear and descriptive, users can find the information they seek more easily, and they
-                  can understand the relationships between different parts of the content more easily.
-                  Descriptive labels help users identify specific components within the content.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ウェブページにどんな情報があるのか、及びその情報がどのように構成されているのかを、利用者が理解するのを手助けすることである。見出しが明快かつ内容が分かるように記述されている場合、利用者は自分の探している情報をより容易に見つけることができ、コンテンツ内の様々な部分間の関係をより容易に理解できる。内容が分かるように記述されているラベルは、利用者がコンテンツ内にある特定のコンポーネントを識別するのに役立つ。
                   
                </p>
-               <p>Labels and headings do not need to be lengthy. A word, or even a single character,
-                  may suffice if it provides an appropriate cue to finding and navigating content.
+               <p>ラベル及び見出しを長くする必要があるわけではない。単語一つ又は一文字だけであっても、コンテンツを見つけてナビゲートする手がかりとして適切であれば、それだけで十分なこともある。
                   
                </p>
-               <p>This Success Criterion does not require headings or labels. This Success Criterion
-                  requires that if headings or labels are provided, they be descriptive. This Success
-                  Criterion also
-                  does not require that content acting as a heading or label be correctly marked up
-                  or
-                  identified – this aspect is covered separately by
-                  <a href="info-and-relationships">1.3.1: Info and Relationships</a>. It is possible for content
-                  to pass this Success Criterion (providing descriptive content that acts as headings
-                  or labels) while failing
-                  Success Criterion 1.3.1 (if the headings or labels aren't correctly marked up/identified).
-                  Conversely,
-                  it is also possible for content to pass Success Criterion 1.3.1 (with headings or
-                  labels correctly
-                  marked up or identified), while failing this Success Criterion (if those headings
-                  or labels are not
-                  sufficiently clear or descriptive).
-                  
+               <p>この達成基準は見出しやラベルを必要としない。見出しやラベルが提供されている場合、それらの内容が分かるように記述されていることが求められる。この達成基準は見出し又はラベルとして機能するコンテンツを正しくマークアップすること又は識別することも必要としない。この点は <a href="info-and-relationships">1.3.1: 情報及び関係性</a>で別途取り扱っている。コンテンツがこの達成基準に合格しながら (見出し又はラベルとして機能する説明的なコンテンツを提供している)、達成基準 1.3.1 (見出し又はラベルを誤ってマークアップ / 識別している場合) に失敗することがありえる。反対に、コンテンツが達成基準 1.3.1 に合格しながら (見出し又はラベルが正しくマークアップ又は識別されている)、この達成基準に失敗することも可能である (それらの見出し又はラベルが十分に明確又は説明的ではない場合)。
```

Comment:
次の段落で「失敗するということもありえる」としている箇所と同じ It is possible... がこちらにもありましたので揃えたいです。

```suggestion
               <p>この達成基準は見出しやラベルを必要としない。見出しやラベルが提供されている場合、それらの内容が分かるように記述されていることが求められる。この達成基準は見出し又はラベルとして機能するコンテンツを正しくマークアップすること又は識別することも必要としない。この点は <a href="info-and-relationships">1.3.1: 情報及び関係性</a>で別途取り扱っている。コンテンツがこの達成基準を満たしながら (見出し又はラベルとして機能する説明的なコンテンツを提供している)、達成基準 1.3.1 (見出し又はラベルを誤ってマークアップ / 識別している場合) に失敗することがありえる。反対に、コンテンツが達成基準 1.3.1 を満たしながら (見出し又はラベルが正しくマークアップ又は識別されている)、この達成基準には失敗するということもありえる (それらの見出し又はラベルが十分に明確又は説明的ではない場合)。
```